### PR TITLE
core/test: add GenerateTestAddrs test

### PR DESCRIPTION
**Pull request by tbal999**
feel free to adjust this where necessary

this PR adds 1 table test to the core/test section of the repo

### CODE REVIEW

**Comment by marten-seemann** [comment]
What’s motivating this PR? These functions are just used in tests.

**Comment by tbal999** [comment]
just thought "why not"

File: core/test/addrs_test.go [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+			inputInt:       0,
+			expectedOutput: []string{},
+		},
+		{
+			description: "generate a few results",
+			inputInt:    3,
+			expectedOutput: []string{
+				"/ip4/1.2.3.4/tcp/0",
+				"/ip4/1.2.3.4/tcp/1",
+				"/ip4/1.2.3.4/tcp/2",
```
Comment (marten-seemann) [inline core/test/addrs_test.go:26]: This assertion doesn't really make sense. The function would still be valid if the port numbers were different.

File: core/test/addrs_test.go [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+	for _, tt := range tableTest {
+		tt := tt
+
+		t.Run(tt.description, func(t *testing.T) {
+			actualOutput := GenerateTestAddrs(tt.inputInt)
+
+			outputStringArray := []string{}
+
+			for index := range actualOutput {
+				outputStringArray = append(outputStringArray, actualOutput[index].String())
```
Comment (marten-seemann) [inline core/test/addrs_test.go:40]: There's an `Equal` function on multiaddrs.

**Review by marten-seemann — CHANGES_REQUESTED** [review]
I'm still not convinced we actually need this, to be honest.

**Comment by tbal999** [comment]
If you are not convinced, feel free to close this.

**Comment by p-shahi** [comment]
Thanks @tbal999 , going to close this. There are some other issues labled `good first issue` in case you'd like to contribute, we'd be happy to review.

**Pull request closed without merging**